govcd/openapi.go

```diff
@@ -689,6 +689,33 @@ func (client *Client) newOpenApiRequest(apiVersion string, params url.Values, me
 	return req
 }
 
+// OpenApiTestConnection Tests a connection, including SSL handshake and hostname verification.
+func (client *Client) OpenApiTestConnection(testConnection types.TestConnection) (*types.TestConnectionResult, error) {
```

Comment:
In fact I think this function could live in `api.go` with other `client` functions and it also could work without the `OpenApi` prefix (just `TestConnection`). The reason is that it is not really a function verifying OpenApi, but checking VCD access/readyness in general.

The `openapi.go` contains mainly the tools to interact with OpenApi itself.

Reply:
Already moved `OpenApiTestConnection` from openapi.go to api.go. Also renamed to `TestConnection`